spring-cloud-stream-binders/spring-cloud-stream-binder-rabbit/src/main/java/org/springframework/cloud/stream/binder/rabbit/RabbitMessageChannelBinder.java

```diff
@@ -641,19 +460,19 @@ private RabbitTemplate determineRabbitTemplate(RabbitPropertiesAccessor properti
 	}
 
 	private Binding<MessageChannel> doRegisterProducer(final String name, MessageChannel moduleOutputChannel,
-			AmqpOutboundEndpoint delegate, RabbitPropertiesAccessor properties) {
+													   AmqpOutboundEndpoint delegate, ProducerProperties properties) {
```

Comment:
May be better to use `RabbitProducerProperties` instead of just the `ProducerProperties` (thought it looks sufficient).